pallets/drand/src/lib.rs

```diff
@@ -55,17 +74,17 @@ mod benchmarking;
 pub mod weights;
 pub use weights::*;
 
-pub mod bls12_381;
-pub mod utils;
-
-const USAGE: ark_scale::Usage = ark_scale::WIRE;
-type ArkScale<T> = ark_scale::ArkScale<T, USAGE>;
-
 /// the main drand api endpoint
-pub const API_ENDPOINT: &str = "https://api.drand.sh";
+pub const API_ENDPOINT: &str = "https://drand.cloudflare.com";
```

Comment:
We should move the endpoint to the config. I have created a follow-up issue for that
```suggestion
// TODO: move to config https://github.com/ideal-lab5/pallets/issues/28
pub const API_ENDPOINT: &str = "https://drand.cloudflare.com";
```